fuzz/src/msg_targets/gen_target.sh

```diff
@@ -7,35 +7,33 @@ GEN_TEST() {
 	echo "pub mod $tn;" >> mod.rs
 }
 
-echo "mod utils;" > mod.rs
+echo "mod utils;
+" > mod.rs
```

Comment:
I thought we were going to stop formatting .sh files?

Reply:
This file I switched up so that the code it generates (in mod.rs) corresponds to rustfmt